app/src/androidTest/java/com/github/mateo762/myapplication/AboutTest.kt

```diff
@@ -0,0 +1,104 @@
+package com.github.mateo762.myapplication
+
+import android.util.Log
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.mateo762.myapplication.about.AboutActivity
+import com.google.firebase.database.ktx.database
+import com.google.firebase.ktx.Firebase
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+@HiltAndroidTest
+class AboutTest {
+
+    private var user_count: String = "Loading..."
+    private var posts_count: String = "Loading..."
+    private var completed_count: String = "Loading..."
+
+    @get:Rule
+    val hiltRule = HiltAndroidRule(this)
+
+    @get:Rule
+    val activityRule = ActivityScenarioRule(AboutActivity::class.java)
+
+
+    @Before
+    fun setUp() {
+        hiltRule.inject()
+
+        val db = Firebase.database.reference
+        db.child("usernames").get().addOnSuccessListener {
+            Log.d("firebase", "Got value ${it.value}")
```

Comment:
The TAG should be the class where you are logging, "firebase" is too broad, for example if you this exactly line in other fragment/activity, you will not know where it happens